src/DeepCopy/DeepCopy.php

```diff
@@ -85,11 +85,33 @@ public function copy($value)
         return $this->recursiveCopy($value);
     }
 
+    /**
+     * @param Filter  $filter
+     * @param Matcher $matcher
+     *
+     * @return void
+     */
     public function addFilter(Filter $filter, Matcher $matcher): void
     {
         $this->filters[] = [$matcher, $filter];
     }
 
+    /**
+     * Add filter with high priority
+     *
+     * @param Filter  $filter
+     * @param Matcher $matcher
+     *
+     * @return void
+     */
+    public function prependFilter(Filter $filter, Matcher $matcher): void
+    {
+        \array_unshift($this->filters, [
```

Comment:
I would rather add a `use function array_unshift;` statement at the top of the file than having this leading backslash. The leading backslash is necessary in older version of PHP as `use function` is not allowed there, but in this case we do not support those old versions. So it's more a matter of style and I find the use statement more readable

Reply:
I made mistake with branches, i think i made PR to 1.8. Damn, sorry for that.
I added unit tests and fix prependFilter method for 2.x version, now it's OK.